.github/workflows/ci.yml

```diff
@@ -9,8 +9,8 @@ jobs:
     strategy:
       matrix:
         version:
-          - '1.5'
           - '1.6'
+          - '1'
```

Comment:
Looking at when it was added in Javis, I believe it checks for 1.7, so now we should have tests on 1.6 (LTS) and 1.7.smth latest with these lines. Though I might be wrong here.

Reply:
No problem!
Yea, it checks for the latest stable release from Julia which is 1.7 so that works.
We will just need to bump in the future (which is what we do manually for Javis anyways).
Looks good!